Annotation/declarative_golden_set.json

```diff
@@ -402,7 +426,7 @@
                     }

```

Comment:
I think I do agree with not including "Jones to be a person with inherent dignity", but think we should bring it up with supervisors?

Reply:
I think it's pretty clear to me that it's not part of the embedded clausal complement, the thing which is seen, so I would see it as necessary to bring up. But I'd be happy to do so if you'd like